frontend/package.json

```diff
@@ -40,5 +40,30 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.27.0",
+    "@typescript-eslint/parser": "^2.27.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-typescript": "^7.2.0",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.19.0",
+    "eslint-plugin-react-hooks": "^2.5.1",
+    "husky": "^4.2.3",
+    "lint-staged": "^10.1.3",
+    "prettier": "2.0.4"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
Husky does not play well with custom pre-commit .git/hook from the other PR.
It does not install properly if a hook already exists, I had to wipe the .git/hooks directory and reinstall husky to get it to work, but then the back end hook is no longer in place.

Can we just add the linters to the existing .githooks/pre-commit file instead?

Reply:
Wasn't aware of this, I'll go ahead and start switching over to the github hooks for the pre commit checks.

Reply:
OK. It's probably a good idea to have a single source of truth for the git hooks anyway.